timemachine/md/builders.py

```diff
@@ -223,9 +246,35 @@ def get_host_coords():
 
     assert modeller.getTopology().getNumAtoms() == solvated_host_coords.shape[0]
 
-    system = ff.createSystem(modeller.getTopology(), nonbondedMethod=app.NoCutoff, constraints=None, rigidWater=False)
+    omm_host_system = ff.createSystem(
+        modeller.getTopology(), nonbondedMethod=app.NoCutoff, constraints=None, rigidWater=False
+    )
     # Determine box from the system's coordinates
     box = get_box_from_coords(solvated_host_coords)
 
-    # TODO: minimize the water box (BFGS or scipy.optimize)
-    return system, solvated_host_coords, box, modeller.getTopology()
+    (bond, angle, proper, improper, nonbonded), masses = openmm_deserializer.deserialize_system(
```

Comment:
nit/optional: could improve safety by modifying `deserialize_system` to return `tuple[BoundPotential[HarmonicBond], BoundPotential[HarmonicAngle], ...]` (that way it'd be a mypy error to get these in the wrong order)

Reply:
I will do this one separately - along with the HostConfig typing change. I think it makes more sense to do most of them at the same time given how coupled they are.